src/models/StyleTags.js

```diff
@@ -305,6 +305,11 @@ const makeBrowserTag = (
       // $FlowFixMe
       getImportRuleTag().insertRules(`${id}-import`, importRules)
     }
+
+    if (marker.replaceWholeText) {
```

Comment:
It's a deprecated method for replacing content of the text node.  It was removed from all modern browsers.

Reply:
Actually I think i'd like to move this up into the `else` portion of the loop so it's more clear what this is an alternative to. @kitten do you remember why we are using `appendData` vs a more mainstream DOM API call?

Reply:
@probablyup It’s not an alternative of ‘appendData’, it just says to IE that the node was updated and we don’t need to do it in the loop.

Reply:
@probablyup i think this stuff is quite old and it’s only used in development so it didn’t really matter. No time was spent to investigate alternatives 😛

Reply:
So reading through this some things need to be documented:

- where does “replaceWholeText” stem from?
- what does this force IE to do? “Update text node” is inaccurate

That’d be great 👍 also, have your checked that “appendChild” here never appends the child twice in both positions? That’d probably break a lot of ordering guarantees.

Reply:
@Anber could you take a look at these? ^

Reply:
Yes, I had checked appendChild behavior before.
I’ll add documentation in two weeks when I return from vacation :)

Reply:
@kitten, I've added a comment. I hope, it has become more clear :)